lib/swaps/Swaps.ts

```diff
@@ -613,7 +614,7 @@ class Swaps extends EventEmitter {
         return rPreimage;
       } else {
         // we need to get the preimage by making a payment
-        const swapClient = this.swapClients.get(currency);
+        const swapClient = this.swapClientManager.get(currency);
```

Comment:
Same here for try/catch if the swap client doesn't exist, replacing the if statement below.

Reply:
Why do we need to try catch here if the `SwapClientManager.get` will throw a valid error if the swap client doesn't exist?